docs/guides/edgemesh_test_env_guide.md

```diff
@@ -12,12 +12,40 @@ edged:
 - install CNI plugin
 
 # Edgemesh test env config guide
+## Docker support (probation)
+### Configure the edge.yaml using docker
++ Configure the following fields in kubeedge/edge/conf/edge.yaml
+```yaml
+edged:
+    runtime-type: docker
+    remote-runtime-endpoint: unix:///var/run/dockershim.sock
+    remote-image-endpoint: unix:///var/run/dockershim.sock
+    cgroup-driver: cgroupfs
+```
++ make sure the cgroup-driver is same as the configration of docker which can be found by shell command "docker info"
+  + that's to say the value of cgroup-driver is depending on the configration of docker
+```yaml
+    cgroup-driver: cgroupfs
+```
+### Install Docker
```

Comment:
A question here :), does the docker and containd haven't installed when edge have installed? And why we need install them in the mesh guide...

Reply:
yes, I will modify it here, leaving only the network configuration part related to containerd